addons/point_of_sale/tools/posbox/configuration/connect_to_server_wifi.sh

```diff
@@ -58,67 +58,92 @@ function connect () {
 	# make network choice persistent
 	if [ -n "${ESSID}" ] ; then
 		if [ -n "${PERSIST}" ] ; then
-			logger -t posbox_connect_to_wifi "Making network selection permanent"
+			logger -t posbox_connect_to_server_wifi "Making network selection permanent"
 			sudo mount -o remount,rw /
 			echo "${ESSID}" > ${PERSISTENT_WIFI_NETWORK_FILE}
 			echo "${PASSWORD}" >> ${PERSISTENT_WIFI_NETWORK_FILE}
 			sudo mount -o remount,ro /
 		fi
 	else
-		logger -t posbox_connect_to_wifi "Reading configuration from ${PERSISTENT_WIFI_NETWORK_FILE}"
+		logger -t posbox_connect_to_server_wifi "Reading configuration from ${PERSISTENT_WIFI_NETWORK_FILE}"
 		ESSID=$(head -n 1 "${PERSISTENT_WIFI_NETWORK_FILE}" | tr -d '\n')
 		PASSWORD=$(tail -n 1 "${PERSISTENT_WIFI_NETWORK_FILE}" | tr -d '\n')
 	fi
 
 	echo "${ESSID}" > ${CURRENT_WIFI_NETWORK_FILE}
 	echo "${PASSWORD}" >> ${CURRENT_WIFI_NETWORK_FILE}
 
-	logger -t posbox_connect_to_wifi "Connecting to ${ESSID}"
+	logger -t posbox_connect_to_server_wifi "Connecting to ${ESSID}"
 	sudo service hostapd stop
-	sudo killall nginx
-	sudo service nginx restart
-	sudo service dnsmasq stop
 
-	sudo pkill wpa_supplicant
-	sudo ifconfig wlan0 down
-	sudo ifconfig wlan0 0.0.0.0  # this is how you clear the interface's configuration
-	sudo ifconfig wlan0 up
-
-	if [ -z "${PASSWORD}" ] ; then
-		sudo iwconfig wlan0 essid "${ESSID}"
+	# Do we have to use the NetworkManager ?
+	NM=false
+	current_iotbox_version=$(cat "/var/odoo/iotbox_version")
+	required_version="23.11"
+	if [[ "$current_iotbox_version" == "$required_version" ]] || [[ "$current_iotbox_version" > "$required_version" ]]; then
```

Comment:
Why not `>=` ?
https://www.baeldung.com/linux/bash-compare-variables-numbers

Reply:
Because that's not numbers but strings

Reply:
Ok
you can just compare from another version and so keep only the ">"
so compare from 23.09